groundstation/static/js/App.jsx

```diff
@@ -169,7 +184,15 @@ function App() {
                         <ClickAwayListener onClickAway={handleClose}>
                           <MenuList autoFocusItem={open} id="menu-list-grow" onKeyDown={handleListKeyDown}>
                             <MenuItem disabled>{username}</MenuItem>
-                            <MenuItem onClick={handleClose}>
+                              {isAdmin && 
+                              <MenuItem onClick={handleClose}>
+                                <a style={{color: 'rgb(40, 50, 76)'}}href="/adduser">Add User</a>
+                              </MenuItem> }
```

Comment:
```suggestion
```
Remove this button to declutter the dropdown as we already have a button for it in manage users